src/images/badges/fixed-bugs.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please set the font color to white to work more consistently with other badges.

![image](https://user-images.githubusercontent.com/8878152/92506765-f04a0e00-f1d3-11ea-9a41-688c0bdd96a5.png)